react-app/src/api/index.js

```diff
@@ -0,0 +1,86 @@
+import Account from 'account';
+import { createEntry } from 'utils/entry';
+
+const _HOST = 'http://localhost/';
+
+class ErrorCannotGetEntries extends Error {}
+
+class Api {
+  initAccount(username, keyPair) {
+    this._account = new Account(username, keyPair);
+  }
+
+  _getUrl(path) {
+    let url = `${_HOST}${path}`;
+    if (this._token) url += `?token=${this._token}`;
+
+    return url;
+  }
+
+  async _get(path) {
+    const url = this._getUrl(path);
+
+    return await fetch(url);
```

Comment:
Using `return await` is redundant here.
see this explanation https://jakearchibald.com/2017/await-vs-return-vs-return-await/
Maybe we should add an eslint rule for it, https://github.com/eslint/eslint/blob/master/docs/rules/no-return-await.md

In any case, this is not an important problem and we could easily ignore it.

Reply:
I missed this... :/

Reply:
Done.